custom_model_runner/README.md

```diff
@@ -138,6 +138,40 @@ Null value imputation   PASSED
 ```
 In case of check failure more information will be provided.
 
+### Runtime Parameters
+The runtime parameters are parameters that the user creates via the DataRobot WEB UI, under the
```

Comment:
Runtime parameters are created by the user via ... ROUTE NAME.  These runtime parameters can then be loaded into the custom model as follows.